app/src/layouts/DashboardLayout/DashboardContent.js

```diff
@@ -220,9 +220,8 @@ const DashboardContent = () => {
           {onboardingVariation !== "variant1" &&
             shouldShowOnboarding() &&
             appMode !== GLOBAL_CONSTANTS.APP_MODES.DESKTOP &&
-            !appOnboardingDetails.isOnboardingCompleted && (
-              <Onboarding isOpen={activeModals.appOnboardingModal.isActive} />
-            )}
+            !appOnboardingDetails.isOnboardingCompleted &&
+            !isAppTypeSessionBear() && <Onboarding isOpen={activeModals.appOnboardingModal.isActive} />}
```

Comment:
just add the check inside `shouldShowOnboarding` in [`features/onboarding/utils.ts`](https://github.com/requestly/requestly/blob/fa6e8d683428a3973f172e57f86bed6b2f1803b7/app/src/features/onboarding/utils.ts)